index.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const knex = require('knex')({
+    client: 'mysql',
+    connection: {
+        host: '127.0.0.1',
+        port: 3306,
+        user: 'root',
+        password: 'my-secret-pw',
+        database: 'test'
+    }
+});
+
+const mealsRouter = require('./routers/meals');
+const reservationsRouter = require('./routers/reservations');
+
+const app = express();
+const port = 3000;
+
+const sendJSON = (res, status, data) => {
+    res.status(status).json(data);
+};
+
+app.use(express.json());
+
+app.use('/api/meals', mealsRouter);
+app.use('/api/reservations', reservationsRouter);
+
+app.get('/test-connection', async (req, res) => {
+    try {
+        const result = await knex.raw("SELECT 1 + 1 AS result");
+        sendJSON(res, 200, { message: "Database connection is working!", result: result[0] });
+    } catch (error) {
+        console.error("Error testing database connection:", error);
+        sendJSON(res, 500, { error: "Database connection failed." });
+    }
+});
+
+app.get('/future-meals', async (req, res) => {
+    try {
```

Comment:
Whenever you use try catch, you want to encapsulate the least amount of code that you suspect that can fail, that is try to not have every line of code inside the try catch. In this example the knex query can fail, but sendJSON is also inside the try catch. Debugging code where multiple statements are inside the try catch makes it difficult to find bugs in the future.